test/integration/services/RabbitMQService.test.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+
+const AMQPStats = require('amqp-stats');
+const should = require('should');
+
+/**
+ * RabbitMQ connection.
+ */
+describe('RabbitMQ', () => {
+  /**
+   * Helper: get default client.
+   */
+  function getManagementClient() {
+    return new AMQPStats({
+      username: sails.config.amqp.management.username,
+      password: sails.config.amqp.management.password,
+      hostname: sails.config.amqp.management.hostname,
+      protocol: sails.config.amqp.management.protocol,
```

Comment:
That's a RabbitMQ management client, it doesn't connect to a particular vhost.
It's possible to specify vhost while querying for stats: https://github.com/timisbusy/node-amqp-stats/